FWCore/ParameterSet/python/Config.py

```diff
@@ -727,18 +839,17 @@ def dumpPython(self, options=PrintOptions()):
         result+=self._dumpPythonList(self.filters_() , options)
         result+=self._dumpPythonList(self.analyzers_(), options)
         result+=self._dumpPythonList(self.outputModules_(), options)
-        result+=self._dumpPythonList(self._sequencesInDependencyOrder(), options)
-        result+=self._dumpPythonList(self.paths_(), options)
-        result+=self._dumpPythonList(self.endpaths_(), options)
         result+=self._dumpPythonList(self.services_(), options)
         result+=self._dumpPythonList(self.es_producers_(), options)
         result+=self._dumpPythonList(self.es_sources_(), options)
         result+=self._dumpPython(self.es_prefers_(), options)
+        result+=self._dumpPythonList(self._tasksInDependencyOrder(), options)
+        result+=self._dumpPythonList(self._sequencesInDependencyOrder(), options)
+        result+=self._dumpPythonList(self.paths_(), options)
+        result+=self._dumpPythonList(self.endpaths_(), options)
         result+=self._dumpPythonList(self.aliases_(), options)
-        if self.schedule:
-            pathNames = ['process.'+p.label_() for p in self.schedule]
-            result +='process.schedule = cms.Schedule(*[ ' + ', '.join(pathNames) + ' ])\n'
-
+        if not self.schedule_() == None:
+            result += 'process.schedule = ' + self.schedule.dumpPython(options)
```

Comment:
Yes. Just to be sure I ran a test and it works. If I dump a process with more than 255 paths that works and I can also execute the configuration created by dumpPython in that case.